Prevoker/Disc/HPal Ability Updates

yurikenus opened the pull request:
- **Discipline Priest:** added Atonement aura, Harsh Discipline & stacks,  and Power Word Radiance charges with talent.
- **Holy Paladin:** added Maraad's Dying Breath aura.
- **Preservation Evoker:** added better Stasis handling, Rewind talent updates, and updates to support interactions with Verdant Embrace, Lifebind, and Call of Ysera.

yurikenus committed:
Prevoker/Disc/HPal Ability Updates

Discipline Priest: added Atonement aura, Harsh Discipline & stacks,  and Power Word Radiance charges with talent.
Holy Paladin: added Maraad's Dying Breath aura.
Preservation Evoker: added better Stasis handling, and updates to support interactions with Verdant Embrace, Lifebind, and Call of Ysera.

Hekili commented:
1. Are these for 10.0.7 changes or 10.0.5?  You committed to the current branch, which is still 10.0.5.
2. Please use code formatting consistent with the rest of the addon's code (spacing, bracing, etc.).
3. Don't set an auras `stack` to 0; remove the buff if it is consumed.

I'll take a closer look later.  Thanks!

yurikenus committed:
Updated Disc - Harsh Discipline Buff Removal

Based on comments, changing spacing and removing the buff (rather than setting stacks to zero).

yurikenus committed:
More spacing updates

yurikenus commented:
Hey Hekili - thanks for checking.  Here's some answers:

1. These are for 10.0.5, but should also work for 10.0.7.  There's no rush to squeeze them in early, if you want we can push them into 10.0.7 instead for next week.  Whatever you like.
2. Sorry, I have had other PRs that were corrected to remove some spaces some times, so I thought you were gradually trying to change the format.  I updated the Disc and Prevoker ones to put in the standard format I see most places.  If I'm doing it wrong, just let me know how you want it to be and I'll make future PRs match.
3. I fixed the buff stuff, bad logic on my end, thanks for the note.

Hekili committed:
Update EvokerPreservation.lua

Hekili committed:
Update EvokerPreservation.lua

Hekili committed:
Update EvokerPreservation.lua

Hekili committed:
Update PaladinHoly.lua

Hekili committed:
Update PriestDiscipline.lua
